site/gatsby-site/content/research.mdx

```diff
@@ -31,7 +31,7 @@ Every incident has its own suggested citation that credits both the submitter(s)
 
 **Add Single Incident Reports**
 
-We currently accept incident reports in a [quick add](/apps/quickadd) form, which does not credit submitters, and the [submit app](/apps/submit) which allows you to assign credit for the submission. Both types of submissions are a great help to preventing repeating history.
+We currently accept incident reports in a [quick add](/apps/submit) form, which does not credit submitters, and the [submit app](/apps/submit) which allows you to assign credit for the submission. Both types of submissions are a great help to preventing repeating history.
```

Comment:
`We currently accept incident reports in a [quick add](/apps/submit) form` -> `We currently accept incident reports in a quick add form on the [landing page](/)`